### PR TITLE
High-Capacity Laser Battery Point Increase

### PR DESCRIPTION
**Pull request by Geevies**
## About The Pull Request

Increases the cost of HC Laser Batteries for CAS to 200 (up from 100).

## Why It's Good For The Game

Brings the cost of the ammo in line with other CAS ammunition, and makes other CAS weaponry more viable by doing so.

## Changelog

:cl:
balance: Increases the cost of HC Laser Batteries for CAS to 200 (up from 100).
/:cl:

### CODE REVIEW

**Comment by FoxyStalin** [comment]
'Makes other Weapons more viable'  as if Lasers isn't the second worst option other than GAU (Until the GAU Buff gets in)

**Comment by c4xmaniac** [comment]
Laser killer PR, time of the Xwing is over. It was only viable due to how much you could use because it’s not particularly strong.

File: code/modules/cm_marines/dropship_ammo.dm [hunk 1]
```diff
@@ -201,7 +201,7 @@
 	ammo_used_per_firing = 10
 	max_inaccuracy = 1
 	warning_sound = 'sound/effects/nightvision.ogg'
-	point_cost = 100
+	point_cost = 250
```
Comment (stanalbatross) [inline code/modules/cm_marines/dropship_ammo.dm:204]:
```suggestion
	point_cost = 200
```
doesn't need to be 250

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Review by stanalbatross — APPROVED** [review]

**Comment by Weuyn** [comment]
The laser having such cheap ammo is a relic of the time the hangar had a bunch of ammo depots for every other weapon type lying around.

**Pull request merged**